_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -0,0 +1,288 @@
+		}
+	</style>
+	<body>
+		<p style="line-height: 20px;">
+			The boy walked down the street in a carefree way, playing without notice of what was about him. He didn't hear the
+			sound of the car as his ball careened into the road. He took a step toward it, and in doing so sealed his fate.
+		</p>
+	</body>
+</html>
+```
+
+#### Passed Example 2
+
+This `div` element has a `letter-spacing` of `0.12em` which is equal to the permissible minimum, given the pixel size of the body is 16 pixels. Since the [important flag][] is not set, the user can increase the `letter-spacing` above the permissible minimum value.
+
+```html
+<html>
+	<style>
+		body {
+			font-size: 16px;
+		}
+	</style>
+	<body>
+		<div style="letter-spacing: 0.12em;">
+			The boy walked down the street in a carefree way, playing without notice of what was about him. He didn't hear the
+			sound of the car as his ball careened into the road. He took a step toward it, and in doing so sealed his fate.
+		</div>
+	</body>
+</html>
+```
+
+#### Passed Example 3
+
+This `strong` element has a `word-spacing` of `1.92pt !important` (equals `0.12em`) which is equal to the permissible value, given the pixel size of the body is 16 pixels.
+
+```html
+<html>
+	<style>
+		body {
+			font-size: 16px;
+		}
+	</style>
+	<body>
+		<strong style="word-spacing: 1.92pt !important;">
+			The boy walked down the street in a carefree way, playing without notice of what was about him. He didn't hear the
+			sound of the car as his ball careened into the road. He took a step toward it, and in doing so sealed his fate.
+		</strong>
+	</body>
+</html>
+```
+
+#### Passed Example 4
+
+This `article` element has both `word-spacing` and `line-height` specified, which are both below the permissible minimum, given the pixel size of the body is 16 pixels. Since the [important flag][] is not set for either the `line-height` or `word-spacing`, the user can increase them to be above the permissible minimum value.
+
+```html
+<html>
+	<style>
+		body {
+			font-size: 16px;
+		}
+	</style>
+	<body>
+		<article style="word-spacing: 1.8pt; line-height: 1em;">
+			The boy walked down the street in a carefree way, playing without notice of what was about him. He didn't hear the
+			sound of the car as his ball careened into the road. He took a step toward it, and in doing so sealed his fate.
+		</article>
+	</body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `p` element has a `line-height` of `30px !important` (equals `1.875em`) which is above the permissible minimum, given the pixel size of the body is 16 pixels.
+
+```html
+<html>
+	<style>
+		body {
+			font-size: 16px;
+		}
+	</style>
+	<body>
+		<p style="line-height: 30px !important">
+			The boy walked down the street in a carefree way, playing without notice of what was about him. He didn't hear the
+			sound of the car as his ball careened into the road. He took a step toward it, and in doing so sealed his fate.
+		</p>
+	</body>
+</html>
```

Comment:
I think Jean-Yves is correct, this example would pass the SC and the expectations of this rule.